apps/site/assets/ts/schedule/components/schedule-finder/daily-schedule/DailyScheduleSubway.tsx

```diff
@@ -171,15 +165,19 @@ const DailyScheduleSubway = ({
         >
           <div className="m-schedule-page__sidebar-hours">
             <div className="font-weight-bold fs-14">Regular schedule</div>
-            <div className="fs-16 pt-8">{`Trains depart every ${scheduleNoteText}`}</div>
+            <div className="fs-16 pt-8">
+              {scheduleNoteText !== "" &&
```

Comment:
This will show an empty accordion; I think it would be a better user experience to hide the `<ExpandableBlock />` entirely if `scheduleNoteText !== ""`.